.pre-commit-config.yaml

```
@@ -0,0 +1,9 @@
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.2.3
+  hooks:
+  - id: trailing-whitespace
+    exclude: &exclude_files >
+      (?x)^(
+        tests/inputs/various-whitespace.filter|
+        tests/inputs/fuzzystring-leading-trailing-spaces.filter
+      )$
```

Not necessary. Just an EOL new line.
```suggestion
      )$

```